src/py/flwr/server/strategy/fedavgm.py

```diff
@@ -59,7 +67,7 @@ def __init__(
         on_fit_config_fn: Optional[Callable[[int], Dict[str, Scalar]]] = None,
         on_evaluate_config_fn: Optional[Callable[[int], Dict[str, Scalar]]] = None,
         accept_failures: bool = True,
-        initial_parameters: Optional[Parameters] = None,
+        initial_parameters: Optional[Parameters],
```

Comment:
```suggestion
        initial_parameters: Optional[Parameters] = None,
```

Reply:
It is preferable to leave as `None` as default. In this way, users can still use this strategy without having to worry about passing a (often randomly initialised) model.